include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -115,144 +115,155 @@ class input_buffer_adapter : public input_adapter_protocol
     const char* const limit;
 };
 
-template<typename WideStringType>
-class wide_string_input_adapter : public input_adapter_protocol
+namespace
```

Comment:
We're already in `namespace detail`, plus having anonymous namespaces in headers can be dangerous, as explained in #552